internal/business/persisted_operations/gcp_storage_loader.go

```diff
@@ -80,5 +88,13 @@ func (g *GcpStorageLoader) Load(ctx context.Context) error {
 		_ = reader.Close()
 	}
 
+	log.Info(fmt.Sprintf("Read %d manifest files from bucket", numberOfFilesProcessed))
+
 	return errors.Join(errs...)
 }
+
+func getFileName(attrs *storage.ObjectAttrs) string {
```

Comment:
Could you add some tests on this or clarify its function with a more elaborate name/docstring? I can't make up what this function does from its body